gframe/duelclient.cpp

```diff
@@ -3357,11 +3357,11 @@ bool DuelClient::ClientAnalyze(unsigned char* msg, int len) {
 		return true;
 	}
 	case MSG_ADD_COUNTER: {
-		int type = BufferIO::ReadInt16(pbuf);
+		int type = buffer_read<uint16_t>(pbuf);
 		int c = mainGame->LocalPlayer(BufferIO::ReadUInt8(pbuf));
 		unsigned int l = BufferIO::ReadUInt8(pbuf);
 		int s = BufferIO::ReadUInt8(pbuf);
```

Comment:
At the very least, the same style should be used within the same block of code.

Reply:
done